pkg/core/helper_test.go

```diff
@@ -213,6 +213,7 @@ func TestCreateBasicChain(t *testing.T) {
 	bw := io.NewBufBinWriter()
 	txSendRaw.EncodeBinary(bw.BinWriter)
 	t.Logf("sendrawtransaction: %s", hex.EncodeToString(bw.Bytes()))
+	require.False(t, saveChain)
```

Comment:
Not sure about the easiest way, but yeah this helps not to forget to change it back to false.